docs/en/operations/table_engines/mysql.md

```diff
@@ -3,25 +3,45 @@
 
 The MySQL engine allows you to perform `SELECT` queries on data that is stored on a remote MySQL server.
 
-Call format:
+Old format:
 
 ```
-MySQL('host:port', 'database', 'table', 'user', 'password'[, replace_query, 'on_duplicate_clause']);
+MySQL(remote_address, remote_database, remote_table_name, user, password[, replace_query, on_duplicate_clause]);
 ```
 
-**Call parameters**
+New format:
 
-- `host:port` — Address of the MySQL server.
-- `database` — Database name on the MySQL server.
-- `table` — Name of the table.
+```
+MySQL SETTINGS
+  remote_address = 'host:port',
+  remote_database = 'database',
+  remote_table_name = 'table',
+  user = 'user',
+  password = 'password'
+  replace_query = 0,
+  on_duplicate_clause = 'on_duplicate_clause',
+  mysql_variable_wait_timeout = 28800
```

Comment:
Can we get rid of this prefix `mysql_variable_`  ? It's quite clear that the settings after `MySQL SETTINGS` are related to `MySQL`.